fab/environments.yml

```diff
@@ -87,90 +77,75 @@ production:
   environment: production
   should_migrate: True
 
-  sms_queue_enabled: True
-  pillow_retry_queue_enabled: True
-  celery_periodic_enabled: True
-  reminder_queue_enabled: True
-  reminder_rule_queue_enabled: True
-  reminder_case_update_queue_enabled: True
-
   new_relic_enabled:
     - hqdjango3.internal.commcarehq.org
 
-  celery_params:
-    main:
-      concurrency: 8
-    background_queue:
-      concurrency: 4
-    periodic:
-      concurrency: 4
-    pillow_retry_queue:
-      concurrency: 1
-    reminder_case_update_queue:
-      concurrency: 4
-    reminder_queue:
-      concurrency: 8
-    reminder_rule_queue:
-      concurrency: 4
-    saved_exports_queue:
-      concurrency: 3
-    sms_queue:
-      concurrency: 8
+  celery_processes:
+    hqcelery0:
```

Comment:
Aha, I see. This is the real change; for the other environments you're just adding `"*"` and indenting.